translation/en/RedirectedResources/assets/abdata/adv/scenario/c-10/00/0/translation.txt

```diff
@@ -1,125 +1,124 @@
-//『作戦って？』=
-//水瀬さんは顔色を変えずに、さらさらとシャーペンを走らせる。=
-//『橋本先生はすっごく厳しいけど、代わりの提案も一緒にするとあまり怒らないんだよ( ..)φ』=
-//なるほど、そういうことだったのか。=
-//『橋本先生はいつも“生徒の自主性を重んじる”って言ってるから、ちゃんと考えてマイナスを埋めようとすると認めてくれるの』=
-//（逆に、単に忘れたことだけ伝えると怒られるってことかな？　もしそうなら、あぶなかった……）=
-//『ありがとう。助かったよ』=
-//『いいんだよ！　それじゃ授業がんばろう(/・ω・)/』=
-//二人で頷き合うと、ちょうど先生が公式を書き終えたところだった。=
-//「これが、今日学ぶ公式よ。次のテストの目玉になるから、この時間でしっかりと要点を抑えるようにね」=
-//（あれ？　この公式、前の学園で習ってたよな。こっちの教科書だと順番が違うのかな）=
-//「２次関数はとてもつまづきやすいところだから、丁寧に教えていくわ」=
-//「わからないところがあったら、何がどうわからないのか考えておいて。後で設問を解く時間を設けるから、そのときに質問しなさい」=
-//橋本先生は式を解いていく順序を説明するプリントを配ったり、実際に自分で問題を解いては解説したりと、本当に丁寧な授業をしてくれた。=
-//（これは……めちゃくちゃわかりやすい！　前の先生もよく教えてくれたけど、橋本先生の丁寧さには適わないかも）=
-//「もう一問解いてみるわね。ちょっとした応用が入ってくるけど、ちゃんと整理すればこんがらがることもないわ」=
-//（学園の先生というよりは塾講師って感じだな……）=
-//そんな感想を抱いていると、隣りの席からノートがすっと差し出された。=
-//『さっきからすごく余裕そうだけど、もしかしてキミって数学できる人？Σ(･ω･ﾉ)ﾉ！』=
-//俺は先生が問題を解いている最中なのを確認してから、筆談を始める。=
-//『そこまでじゃないけど、前の学園でやったところだったから』=
-//『！』=
-//『ごめん、数学ってどうしても苦手でダメダメなんだぁ…… (-_-;)　何かいい方法ないかな(>_<。)』=
-//（う～ん。そうは言ってもな……橋本先生以上にうまく教えるなんて不可能じゃ……）=
-//と思ったところで、前の学園の先生はとにかく教科書の書かれていることに沿って授業をしていたことに気が付いた。=
-//片や橋本先生は、教科書を見るのは設問を確認する時ぐらいで、あとはほとんど自分のノートを見ながら教えている。=
-//（橋本先生の教え方だと無駄を省いている反面、もともと数学がわからない人はもっとわからなくなっちゃうのかな？）=
-//だとしたら、水瀬さんには１から１０までコツコツ教える、前の学園の先生のやり方の方が合っているのかもしれない。=
-//『わかった。じゃあ、ちょっと教科書を貸してね』=
-//先生がもう何問か解く間、俺は教科書通りのやり方をかみ砕いてノートに書き出してみる。=
-//『こんな感じでどうかな。かなり基礎の基礎から書いておいたけど』=
-//そこまで書くと、問題を解き終えた先生が生徒たちに向き直る。=
-//「じゃあ、この公式を使って問１を解いてみて。１０分後に誰かに前に出て解いてもらうからそのつもりでね」=
-//水瀬さんは『頑張ってみる！』と顔で語ると、俺のノートを片手に問１を解き始める。=
-//俺は別のノートを出して、問１に取り掛かる。=
-//「そろそろ出来たかしら。じゃあ……水瀬さん頑張っていたわね。前に出て解いてみて」=
-//「は、はいっ」=
-//水瀬さんは自分のノートを見ながら、チョークで数字を並べていく。=
-//「……正解よ、よく出来たわね。数学は苦手だったはずだけど、ちゃんと予習をしてきたのね。偉いわ」=
-//「あ、いえ、隣りの席に心強い先生がいるもので……解き方を教えてもらっていました」=
-//水瀬さんが照れ臭そうに言うと、橋本先生の鋭い目線が俺へと注がれる。=
-//「あら、私語は聞こえなかったように思うけど」=
-//「はい、手順をノートに書いてもらったんです」=
-//と言って、水瀬さんは俺が書いた手順を先生に見せる。=
-//「私とは違うやり方ね。キミ、これは前の学園の先生の解き方？」=
-//そう言われて、ちょっと橋本先生に対して失礼なことをしてしまったかもしれないという不安がよぎった。=
-//「あ、はい。実は前の学園ではこの単元はもうやってて……」=
-//俺は、出来るだけ失礼にならないように、前の学園の先生と橋本先生の教え方の違いを説明した。=
-//「なるほどね……私の教え方は塾講師みたい、ね。水瀬さん、どうしてこんなに分からなくなるまで私に質問してくれなかったのかしら？」=
-//「あ、えっと、それは……」=
-//「なんてね。苦手意識を植え付けたのは私だもの、そこは責めないわ。それよりキミ、教え方が上手なのね。先生になれるんじゃない？」=
-//「いえ……橋本先生の教え方が上手だったので、ちゃんとかみ砕いて水瀬さんに教えられただけです」=
-//心からの言葉だったのだが、クラス中から尊敬のまなざしとどよめきが起こって、ちょっと恥ずかしくなる。=
-//「でも、ついてこれない子をさらに突き放しちゃうのは欠点ね、ちょっと考えてみるわ」=
-//「ちょっと脱線しちゃったけど、この問いはそのままテストに出すから、復習しておくようにね」=
-//そこでチャイムが鳴り、数学の授業は終了した。=
-//「さっきはありがとう！　おかげで先生に褒められちゃった」=
-//きっと橋本先生にはあまり褒められたことがないのだろう、水瀬さんはエッヘンと胸を張ってご満悦だ。=
-//「ねね、キミさえよかったらなんだけど、これからも勉強を教えてほしいんだけど……ダメ？」=
-//「ダメじゃないけど、数学以外にも苦手な教科があるの？」=
-//「えっと……歴史と、科学と、英語と……」=
-//「それって、国語以外ってことじゃ……」=
-//「えとえと……国語もちょっと、古典が怪しくて……ご、ごめんね！　私、正直に言っておばかなんだよね！　お願い、助けて！」=
-//「い、言うほどじゃないと思うし、俺でよければいつでも教えるよ」=
-//「本当っ？　やったあ、ありがとう～！」=
-//水瀬さんはぴょんぴょん跳ねてくるくる回った。=
-//（無邪気な子だなぁ……でも正直、この話しやすさに助けられてるよな）=
-//水瀬さんは俺が男だからといって距離を取ることもなく、普通に接してくれる女の子の一人だ。=
-//それは水瀬さんのいいところだと思うし、誰からも好かれる所以なんじゃないかな。=
-//（コイカツ部に誘ってみてもいいかもしれないな）=
-//いつか時間のあるときに、コイカツ部の話をしてみよう。=
-//俺は小躍りする水瀬さんを見ながらそう思った。=
-
-亜依=Ai
-麗奈=Reina
-号令係=Class Representative
```

Comment:
These are still missing (also if you could put them back at the end of the file it's easier to track the history)

Reply:
The ones in Ai's story (adv/scenario/c-10) were never gone. They're just where the dumped data put them, which I assume is immediately before they're used.

Easy enough to move them all to the end for consistency, though.